pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/RuntimeSpawner.java

```diff
@@ -69,7 +69,7 @@ public void start() throws Exception {
                 details.getName(), this.instanceConfig.getInstanceId());
 
         runtime = runtimeFactory.createContainer(this.instanceConfig, codeFile,
-                instanceLivenessCheckFreqMs * 1000);
+                instanceLivenessCheckFreqMs / 1000);
```

Comment:
we should also rename "instanceLivenessCheckFreqMs" since it has "Ms" at the end to indicate its in milliseconds.

Reply:
I don't understand your comment here. "instanceLivenessCheckFreqMs" is in milliseconds, and have "ms" at the end is clear. Here we divide it by 1000 to convert it to seconds.